serpextract/search_engines.json

```diff
@@ -1,94 +1,94 @@
 {
-  "1.cz":[
+  "1.cz": [
     {
-      "backlink":"s\/{k}",
-      "charsets":[
+      "backlink": "s/{k}",
+      "charsets": [
         "iso-8859-2"
       ],
-      "params":[
-        "\/s\\\/([^\\\/]+)\/",
+      "params": [
+        "/s\\/([^\\/]+)/",
```

Comment:
What does the removal of escape characters mean in this case? Is it an artifact of black reformatting or is it necessary to work correctly?

Reply:
Those are changes made in the upstream list. I didn't change them. This is just a JSON file. 

Anyway, the escape characters they removed were unnecessary ones.